tests/operators/test_base.py

```diff
@@ -173,5 +175,21 @@ def test_dbt_mixin_add_cmd_flags_run_operator(args, expected_flags):
 
 def test_abstract_dbt_base_operator_append_env_is_false_by_default():
     """Tests that the append_env attribute is set to False by default."""
-    base_operator = AbstractDbtBaseOperator(task_id="fake_task", project_dir="fake_dir")
+    AbstractDbtBase.__abstractmethods__ = set()
+    base_operator = AbstractDbtBase(task_id="fake_task", project_dir="fake_dir")
     assert base_operator.append_env is False
+
+
+def test_abstract_dbt_base_is_not_airflow_base_operator():
+    AbstractDbtBase.__abstractmethods__ = set()
+    base_operator = AbstractDbtBase(task_id="fake_task", project_dir="fake_dir")
+    assert not isinstance(base_operator, BaseOperator)
+
+
+def test_abstract_dbt_base_init_no_super():
```

Comment:
I added a test so that we do not reintroduce back the conflicting `super.init()` call in `AbstractDbtBase`